src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -304,6 +305,17 @@ static DistributeObjectOps Any_DropRole = {
 	.address = NULL,
 	.markDistributed = false,
 };
+
+static DistributeObjectOps Role_Comment = {
```

Comment:
Database_Comment and Role_Comment are now identical. Let's create only one of them and call that one Any_Comment

Reply:
ObjectType is different.Therefore, we still need two different definition